packages/clay-management-toolbar/src/ClayManagementToolbar.js

```diff
@@ -300,6 +300,15 @@ ClayManagementToolbar.STATE = {
 	 */
 	searchActionURL: Config.string(),
 
+	/**
+	 * Method of the search form.
+	 * @default GET
+	 * @instance
+	 * @memberof ClayManagementToolbar
+	 * @type {?(string|undefined)}
+	 */
+	searchFormMethod: Config.string().value('GET'),
```

Comment:
@4lejandrito could you please change this to `Config.oneOf(['GET', 'POST']).value('GET') ?